builder/dockerfile/builder.go

```diff
@@ -212,6 +213,10 @@ func (b *Builder) build(config *types.ImageBuildOptions, context builder.Context
 
 	var shortImgID string
 	for i, n := range b.dockerfile.Children {
+		if i == len(b.dockerfile.Children) {
+			b.isLast = true
```

Comment:
@ehazlett looking at the code it seems suspicious that it's not being used anywhere.